packages/common/src/intl/locales/en/replenishment.json

```diff
@@ -22,11 +22,14 @@
   "label.supplier-name": "Supplier Name",
   "label.supplier-ref": "Supplier Ref",
   "label.min-months-of-stock": "Minimum MOS",
+  "label.hide-stock-over-threshold": "Hide stock over threshold",
```

Comment:
I'm wondering if this would be more helpful if it said `Hide stock over minimum`? That relates it to the 'Minimum MOS' value better than `threshold` maybe

Reply:
yes much better